include/spirv/unified1/NonSemanticVkspReflection.h

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2020 The Khronos Group Inc.
```

Comment:
```suggestion
// Copyright (c) 2024 The Khronos Group Inc.
```

We need to update other copyright dates in this repo also, but we may as well get this one right from the start.

Reply:
I did not remember why I did not put 2024 at the time.
Commit your change reminds me of the reason.:
It breaks the formatting check as this file needs to be the exact one generated from https://github.com/KhronosGroup/SPIRV-Headers/blob/main/tools/buildHeaders/bin/generate_language_headers.py#L35 which has `2020` as the date.